efficientdet/anchors.py

```diff
@@ -304,7 +304,7 @@ def _generate_detections_tf(cls_outputs,
                             image_size,
                             min_score_thresh=MIN_SCORE_THRESH,
                             max_boxes_to_draw=MAX_DETECTIONS_PER_IMAGE,
-                            soft_nms_sigma=0.0,
+                            soft_nms_sigma=1.0,
```

Comment:
I saw the original paper recommends 0.5. Is 1.0 better?

Reply:
https://github.com/tensorflow/tensorflow/issues/40253

Reply:
Oh, interesting. It looks like a bug in TF.

Could you add a comment for this? (maybe just add the link tensorflow/tensorflow#40253).   Also, do you observe better results for the example image using soft_nms?  (I meant this image: https://user-images.githubusercontent.com/11736571/77320690-099af300-6d37-11ea-9d86-24f14dc2d540.png)

Reply:
Hi like I said in my Issue post, I passed 0.25 instead of 0.5 and it gave good results.

Reply:
> Oh, interesting. It looks like a bug in TF.
> 
> Could you add a comment for this? (maybe just add the link [tensorflow/tensorflow#40253](https://github.com/tensorflow/tensorflow/issues/40253)). Also, do you observe better results for the example image using soft_nms? (I meant this image: https://user-images.githubusercontent.com/11736571/77320690-099af300-6d37-11ea-9d86-24f14dc2d540.png)

Hi like I said in my Issue post, I passed 0.25 instead of 0.5 and it gave good results.

Reply:
> Oh, interesting. It looks like a bug in TF.
> 
> Could you add a comment for this? (maybe just add the link [tensorflow/tensorflow#40253](https://github.com/tensorflow/tensorflow/issues/40253)). Also, do you observe better results for the example image using soft_nms? (I meant this image: https://user-images.githubusercontent.com/11736571/77320690-099af300-6d37-11ea-9d86-24f14dc2d540.png)

They're similar.